### ISSUE TITLE
Ability to join segments

### ISSUE THREAD

**Issue by Tritlo** [bhilburn/powerlevel9k#166]
Since #164 splits the status segment into three segments, anyone who wants to use all three will have two (or more?) additional segment separators, which can take up precious space. Could we make it possible to join segments so that they display as one segment?

**Comment by bhilburn** [comment]
I think @Tritlo has an excellent point, here.

Thoughts on how to make this possible in #164, @dritter?

**Comment by dritter** [comment]
Excellent idea! Shouldn't be too hard to do. :+1: 

Only thing is: I would like to do this in a separate PR, to avoid a merge conflict with #165 (and this one should be merged first).

**Comment by bhilburn** [comment]
@dritter - Don't forget you have push access to the project. Why don't you create a new branch on top of the HEAD of `next`, merge in #164, base your new branch on top of it, and then submit a PR to `next` from the second branch?

**Comment by dritter** [comment]
Yeah. Anyway. I have an local integration branch and resolved the conflicts there. :)

**Comment by dritter** [comment]
Okay. I pushed my integration branch as `integration_165_164`. Check it out! :)

**Comment by bhilburn** [comment]
@dritter - Looks awesome! Can you submit a PR against `next` so that we have a logged record of the merge?

**Comment by dritter** [comment]
Here we go. #167 it is.

**Comment by bhilburn** [comment]
Closing since #167 has been merged.
